commander/src/commands/transaction/create/delegate.ts

```diff
@@ -101,12 +101,16 @@ export default class DelegateCommand extends BaseCommand {
 			throw new ValidationError('Enter a valid nonce in number string format.');
 		}
 
-		if (!isValidFee(fee)) {
+		if (Number.isNaN(Number(fee))) {
```

Comment:
do we need this validation if we are going to validate below anyway?
I think `convertLSKToBeddows` and `isValidFee` both have validation so it should be enough

Reply:
`convertLSKToBeddows` doesn't check if it's non-numeric like 'a'